tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,88 @@
+from json import loads
+from time import sleep
+
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_ips(host):
+    """
+    Verify MON IPs updated in ceph.conf
+    Args:
+        host: Installer host
+    """
+    # Get updated MON nodes
+    cmd = "cephadm shell ceph orch ps --daemon-type mon --format json"
```

Comment:
Please use the cli method. 
Cephadm(node).ceph.orch.ps

Reply:
@pranavprakash20, I've used this method but I think there is a problem with this Lib: 
 It uses --demon_type but it has to be --daemon_type. Spelling mismatch there. 

2023-03-08 08:53:42,963 (cephci.test_cluster_view) [INFO] - cephci.Automation.cephci.ceph.ceph.py:1542 - Running command cephadm shell ceph orch ps  --demon_type mon on 10.0.211.245 timeout 600
2023-03-08 08:53:45,550 (cephci.test_cluster_view) [INFO] - cephci.Automation.cephci.tests.cephadm.test_cluster_view.py:32 - List of MON host name:

Reply:
The above method doesn't have any hardcoded arguments, so whatever is passed to it will be considered. Please check whether the args passed to the method are correct or not

Reply:
+1

Reply:
Thanks Pranav, I've used CephAdm(host).ceph.orch.ls(service_type="mon", format="json") this is working as expected.